bsp/nxp/mcx/mcxn/Libraries/drivers/drv_spi.c

```diff
@@ -133,7 +150,7 @@ static rt_ssize_t spixfer(struct rt_spi_device *device, struct rt_spi_message *m
   //  if(message->length < MAX_DMA_TRANSFER_SIZE)
     if(0)
     {
-      //  SPI_MasterTransferBlocking(spi->SPIx, &transfer);
+        LPSPI_MasterTransferBlocking(spi->LPSPIx, &transfer);
```

Comment:
Unused blocking transfer function should be removed. Suggest removing all if(0) section since it will be optimized out.

Reply:
Hi,

this follow existing other NXP SoCs'(those LPCs ;) ) bsp style: still keep the non-dma code while rely on the compiler to optimize out. If someone want to try non-dma, just change the "if (0)" line to "if (1)"

I have no preference, but I just want to unify the style in nxp bsps
Thanks

Reply:
for example: line253@bsp/nxp/lpc/lpc55sxx/Libraries/drivers/drv_spi.c

Reply:
for example: line253@bsp/nxp/lpc/lpc55sxx/Libraries/drivers/drv_spi.c